CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# How to contribute to AML Spec
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+The following is a set of guidelines for contributing to AML Spec:
+
+## Did you find a bug?
+- Ensure the bug was not already reported by searching on GitHub under Issues.
+- If you're unable to find an open issue addressing the problem, open a new one.
+Be sure to include a title, a clear description, as much relevant information as possible and examples if possible.
+
+## Contributing Changes
+- Read and sign the [Contributors License Agreement](#contributors-license-agreement)
```

Comment:
SFDC CLA takes care of enforcing this so either we don't write anything and let the bot handle it or we describe how the bot will be triggered automatically...

Reply:
Great news! I'm going to remove it for now because I don't have a step-by-step explanation of how the bot works or interacts with the contributor. But that would be a nice thing to have. I'm adding it to future changes.